.sonarcloud.properties

```diff
@@ -0,0 +1,9 @@
+# Path to sources
+sonar.sources=.
+sonar.exclusions=**/*test*, **/vendor/**, **/zz_generated.*, **/*.yaml, **/test/**
+
+# Source encoding
+sonar.sourceEncoding=UTF-8
+
+# Exclusions for copy-paste detection
+sonar.cpd.exclusions=**/*test*, **/test/** 
```

Comment:
I am having trouble finding documentation to explain what this setting does. Can you explain it? We don't have any test directories so I am not sure we need the second one, and the first one is already excluded in line 3 it seems?

Reply:
I think so. We add our test files with `filename_test.go` instead of having a dedicated test directory